onboarding-api/src/controllers/user/getUser.ts

```diff
@@ -1,11 +1,26 @@
 import { Request, Response } from 'express'
 import { fetchUser } from '../../utils/fetchUser'
 import { reportHttpError } from '../../utils/httpError'
+import { shuftiProRequest } from '../../utils/shuftiProRequest'
 
 export const getUserController = async (req: Request, res: Response) => {
   try {
     const user = await fetchUser(req.wallet, { suppressError: true })
-    return res.send(user)
+
+    if (!user) {
+      return res.send(user)
+    }
+
+    let status
+
+    if (user.investorType === 'entity' && user.manualKybReference) {
+      status = await shuftiProRequest({ reference: user.manualKybReference }, { path: 'status', dryRun: false })
+    }
```

Comment:
```suggestion
      status = await shuftiProRequest({ reference: user.manualKybReference }, { path: 'status' })
    }
```
don't think you need to pass `dryRun: false` that should be the default